capstone-frontend/src/components/BookSearchTile.jsx

```diff
@@ -1,34 +1,87 @@
-            </BookDescriptionOverlay>
-          </Col>
-          <Col>
-            <div className='center-vertical'>
-              <h2 className='book-title'> {props.book.title} </h2>
-              <p className='book-authors'> {props.book.authors.join(', ')} </p>
-            </div>
-          </Col>
-          <Col md='auto'>
-            <Container className='center-vertical'>
-              <BookListAddDropdown bookLists={props.bookLists} updateBookLists={props.updateBookLists} book={props.book} />
-            </Container>
-          </Col>
-        </Row>
-      </Container>
-    </div>
-  );
+class BookSearchTile extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      hasRead: false,
+      bookObject: {},
+    }
+    fetch(`/api/books?userID=${window.localStorage.getItem('userID')}&gbookID=${this.props.book.id}`)
+      .then(response => response.json()).then(books => {
+        if (books.length === 1) {
+          this.setState({hasRead: books.length === 1, bookObject: books[0]});
+        }
+      })
+      .catch(e => console.log(e));
+  }
+
+  markRead = () => {
+    const readBook = {
+      gbookID: this.props.book.id,
+      userID: window.localStorage.getItem('userID'),
+    }
+    fetch('/api/books', {method:'post', body: JSON.stringify(readBook)})
+        .then(response => response.json())
+        .then(bookObject => this.setState({hasRead: true, bookObject}))
+        .catch(e => console.log(e));
+  }
+
+  markUnread = () => {
+    fetch(`/api/books?id=${this.state.bookObject.id}`, {method: 'delete'})
+        .then(this.setState({hasRead: false, bookObject: {}}))
+        .catch(e => console.log(e));
+  }
+
+  componentDidMount() {
+    fetch(`/api/books?userID=${window.localStorage.getItem('userID')}&gbookID=${this.props.book.id}`)
+      .then(response => response.json()).then(books => {
+        if (books.length === 1) {
+          this.setState({hasRead: books.length === 1, bookObject: books[0]});
+        }
+      })
+      .catch(e => console.log(e));
+  }
+
+  render() {
+    return (
+      <Row className="text-center border m-5 bg-light light-gray-border" key={this.props.book.id} >
+        <Col md={3} className="my-4 p-0 ">
+          <BookDescriptionOverlay book={this.props.book}>
+            <img className="img-fluid book-img-md" src={this.props.book.thumbnailLink} alt={this.props.book.title} />
+          </BookDescriptionOverlay>
+        </Col>
+        <Col className="my-4 p-0">
+          <h2 className="mt-4" id='book-title'> {this.props.book.title} </h2>
+          <StarRatings
+            rating={this.props.book.avgRating}
+            starDimension="40px"
+            starSpacing="10px"
+            starRatedColor="gold" />
+          <p className="my-3" id='book-authors'> {this.props.book.authors.join(', ')} </p>
+        </Col>
+
+        <Col md={3} className="my-4 p-0">
```

Comment:
Should this size change on different screen sizes?

Reply:
Changing it like you mentioned in your comment.